bloggies_backend/src/db.ts

```diff
@@ -1,13 +1,14 @@
 /** Database setup */
 
 import { Client } from "pg";
+require('dotenv').config({ path: require('find-config')('.env') });
 
 let DB_URI;
 
 if (process.env.NODE_ENV === "test") {
-  DB_URI = "postgresql:///learning_circle_test";
+  DB_URI = `postgres://${process.env.DB_USER}:${process.env.DB_PASSWORD}@localhost:5432/learning_circle_test`;
```

Comment:
I think we should put the `port` (5432 here) as a `.env` variable so we can decide what port to use ourselves because an issue will conflict if a developer is already locally hosting psql, which defaults to `port 5432`.

Reply:
That's a great and valid point.

Reply:
makes sense! went ahead and fixed this in the commits below, let me know if there's anything else!